firmware/thruna/BMS/Src/App/App_Accumulator.c

```diff
@@ -453,13 +471,14 @@ bool App_Accumulator_CheckFaults(struct Accumulator *const accumulator, struct T
         App_Accumulator_GetMinVoltage(accumulator, &throwaway_segment, &throwaway_loc) < MIN_CELL_VOLTAGE;
     bool communication_fault = App_Accumulator_HasCommunicationError(accumulator);
 
+    App_CanAlerts_SetFault(BMS_FAULT_BLOWN_CELL_FUSE, blown_fuse_fault);
```

Comment:
We should also add a CAN signal indicating the estimated min. number of cells in parallel, as well as the corresponding cell/segment